### ISSUE TITLE
Gallery items are ordered by oldest first

### ISSUE THREAD

**Issue by xuelongmu** [kdevo/osprey-delight#1]
When adding items to the gallery, I'm seeing that the oldest items (from the date field in markdown file) are listed first when viewing the website. According to `gallery.md`, the latest item should be displayed first:

```
# [date] Specify project publication date - changes order; the latest item will be displayed first.
date: "{{ .Date }}"
```

**Comment by kdevo** [comment]
Thanks for your report.
I will first reproduce the issue and then consider to fix it first in the `dev` branch.

Stay tuned.

**Comment by kdevo** [comment]
I think it's more logical that new items are displayed last, so I simply fixed the comment in the archetype in the [new dev branch](https://github.com/kdevo/osprey-delight/commit/a106983eacecfc4556b82c24b1fbfc09e6539f42#diff-6320434a5eddf31093221488df9c2020) without actually changing the implementation itself.

> tldr: doc was wrong, implementation is correct.

In the future, I plan to make the blog optional, since many original Osprey theme users wanted it.
Check it out, feedback is always welcome.

**Comment by xuelongmu** [comment]
I appreciate the update. My reasoning was that my newest projects are the best ones, since I've become a more experienced developer. Would you mind pointing me to the section of the code where the sorting is done, so I can modify it on my fork? I spent a few minutes looking but couldn't seem to find it (front-end isn't really my area). Thanks!

**Comment by kdevo** [comment]
Sure,

the *reverse chronological order* you wish to use is in fact also [Hugo's default](https://gohugo.io/templates/lists/#default-weight--date--linktitle--filepath).
For the *chronological order* speaks that always putting items to the top leads to more changing visuals, possibly confusing recurring visitors.
Both have pros and cons and I totally understand your use-case. In the future, I will provide an option for that (or better: an alternative way of ordering).

Currently, the [the "reverse chronological order"](https://gohugo.io/templates/lists/#reverse-key-order) is **reversed** in [Line 13 of the gallery partial](https://github.com/kdevo/osprey-delight/blob/7e0228cfa48d9d2b0762a67bc46ebe003bb231cb/layouts/partials/gallery.html#L13):

```go
    {{ $galleryReverse := (where .Site.RegularPages.Reverse "Section" "gallery") }}
```

Pretty much that is a double negation `! (!chronological)` = `chronological`. 
Remove the `.Reverse` in the line and you should be fine - it's just `!chronological` then.


Please also consider giving this project a star if you like it and want to support it. 😄

**Comment by kdevo** [comment]
FYI: I reconsidered my choice above and think it's best to keep Hugo's defaults without changing the natural order.
Like you suggested, new items are now at the top. 

This doesn't mean that a user is bound to order by date only though! As now described in the `gallery.md`, it is also possible to filter by "weight" (built-in), in this sense "items with a high weight fall to the bottom" (the precedence is described short but accurately [in the Hugo docs](https://gohugo.io/templates/lists/#default-weight--date--linktitle--filepath) as mentioned above).

> See [here](https://github.com/kdevo/osprey-delight/blob/dev/layouts/partials/gallery.html#L5) for the change.

**Comment by xuelongmu** [comment]
Wow - I did not expect such a detailed response. Thank you for detailing where to make the change, and for your tip about the weight - I didn't know about either of these!

I'm going to merge your latest changes into my fork. Looking forward to future updates to this project.
